e2e/tests/up/up.go

```diff
@@ -311,6 +315,80 @@ var _ = DevPodDescribe("devpod up test suite", func() {
 				err = f.DevPodUp(ctx, tempDir)
 				framework.ExpectNoError(err)
 			}, ginkgo.SpecTimeout(60*time.Second))
+			ginkgo.It("should start a new workspace with non-root workspace owner and default remoteUser", func(ctx context.Context) {
```

Comment:
An additional tests we can add is to have a custom dockerfile with a vscode user with a UID not-1000 (like 1007 I don't know) and then ensure also that this works and that after the "up" we have a proper vscode user with the right UID

Reply:
Sure. Nice idea!! I will add this test.